common/tasks/rosa/hosted-cp/rosa-hcp-deprovision/rosa-hcp-deprovision.yaml

```diff
@@ -67,13 +67,7 @@ spec:
             exit 0
         fi
 
-        trufflehog filesystem /workspace --only-verified --fail
-        EXIT_CODE=$?
-
-        if [ $EXIT_CODE -ne 0 ]; then
-            echo -e "[ERROR]: Found secrets in artifacts... Container artifacts will not be uploaded to OCI registry due to security reasons."
-            exit 1
-        fi
+        leaktk scan --kind Files --resource /workspace | leaktk-remove-files /workspace
```

Comment:
This is so the secure scanning is backported, correct?

Reply:
I think he can just rebase once PR #76 is merged and this will go away

Reply:
rebase done

Reply:
ack